modules/transactions/src/test/java/org/apache/ignite/internal/tx/storage/state/rocksdb/RocksDbTxStateStorageTest.java

```diff
@@ -43,18 +48,28 @@ public class RocksDbTxStateStorageTest extends AbstractTxStateStorageTest {
     @WorkDirectory
     private Path workDir;
 
+    private final ScheduledExecutorService scheduledExecutor = new ScheduledThreadPoolExecutor(1);
+
+    private final ExecutorService executor = Executors.newFixedThreadPool(1);
+
     @Override
     protected TxStateRocksDbTableStorage createTableStorage() {
         return new TxStateRocksDbTableStorage(
                 1,
                 3,
                 workDir,
-                new ScheduledThreadPoolExecutor(1),
-                Executors.newFixedThreadPool(1),
+                scheduledExecutor,
+                executor,
                 () -> 1_000
         );
     }
 
+    @AfterEach
+    void shutdownExecutors() {
```

Comment:
I'm a little bit worried that we stop the executors before we close the storage, maybe we should override the parent's `AfterEach` instead.

Reply:
fix it